### PR TITLE
helmfile 0.69.0

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/helmfile.rb [hunk 1]
```diff
@@ -34,7 +36,7 @@ def install
     - name: test
     EOS
     system Formula["kubernetes-helm"].opt_bin/"helm", "init", "--client-only"
-    output = '"stable" has been added to your repositories'
-    assert_match output, shell_output("#{bin}/helmfile -f helmfile.yaml repos")
+    system("#{bin}/helmfile -f helmfile.yaml repos")
```
Comment (chenrui333) [inline Formula/helmfile.rb]: Need to find a good way to concatenate multiple lines into one line for assertion test.
```
==> /usr/local/opt/kubernetes-helm/bin/helm init --client-only
Creating /private/tmp/helmfile-test-20190603-68543-1akbwyk/.helm
Creating /private/tmp/helmfile-test-20190603-68543-1akbwyk/.helm/repository
Creating /private/tmp/helmfile-test-20190603-68543-1akbwyk/.helm/repository/cache
Creating /private/tmp/helmfile-test-20190603-68543-1akbwyk/.helm/repository/local
Creating /private/tmp/helmfile-test-20190603-68543-1akbwyk/.helm/plugins
Creating /private/tmp/helmfile-test-20190603-68543-1akbwyk/.helm/starters
Creating /private/tmp/helmfile-test-20190603-68543-1akbwyk/.helm/cache/archive
Creating /private/tmp/helmfile-test-20190603-68543-1akbwyk/.helm/repository/repositories.yaml
Adding stable repo with URL: https://kubernetes-charts.storage.googleapis.com
Adding local repo with URL: http://127.0.0.1:8879/charts
$HELM_HOME has been configured at /private/tmp/helmfile-test-20190603-68543-1akbwyk/.helm.
Not installing Tiller due to 'client-only' flag having been set
==> /usr/local/Cellar/helmfile/0.69.0/bin/helmfile -f helmfile.yaml repos
Adding repo stable https://kubernetes-charts.storage.googleapis.com/
"stable" has been added to your repositories

Updating repo
Hang tight while we grab the latest from your chart repositories...
...Skip local chart repository
...Successfully got an update from the "stable" chart repository
Update Complete.

==> /usr/local/Cellar/helmfile/0.69.0/bin/helmfile -v
```

Comment (fxcoudert) [reply Formula/helmfile.rb]: What is the problem? `assert_match` will match a part of a multi-line string

Comment (chenrui333) [reply Formula/helmfile.rb]: yeah, somehow I have trouble doing that. Can you provide some reference snippet for doing assert_match? That would be great.

Comment (fxcoudert) [reply Formula/helmfile.rb]: Maybe it goes to stderr, and you need to add `2>&1` at the end of your command line

Comment (chenrui333) [reply Formula/helmfile.rb]: let me try it.

Comment (chenrui333) [reply Formula/helmfile.rb]: it works. 💯

Comment (chenrui333) [reply Formula/helmfile.rb]: @fxcoudert Thanks!

**Comment by chenrui333** [comment]
Good for the merge.

**Pull request closed without merging**